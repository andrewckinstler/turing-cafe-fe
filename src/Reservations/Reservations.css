section {
  width: 100%;
  display: flex;
  flex-direction: row;
  flex-wrap: wrap;
  padding: 15px;
  
}