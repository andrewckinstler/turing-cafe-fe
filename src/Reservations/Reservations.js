import React from 'react';
import { ReservationCard } from '../ReservationCard/ReservationCard';

export const Reservations = props => {
  const reservations = props.props.map(res => {
    return <ReservationCard {...res} key={res.id} />
  })

  return (
    <div>
      {reservations}
    </div>
  )
}