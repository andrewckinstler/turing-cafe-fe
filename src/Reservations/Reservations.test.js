import React from 'react';
import { shallow } from 'enzyme'
import { Reservations} from './Reservations';

it('should match a snapshot', () => {
  const mockReservations = [
    {
        "id": 1,
        "name": "Christie",
        "date": "12/29",
        "time": "7:00",
        "number": 12
    },
    {
        "id": 2,
        "name": "Leta",
        "date": "4/5",
        "time": "7:00",
        "number": 2
    }
  ]

  const wrapper = shallow(<Reservations props={mockReservations} />)
  expect(wrapper).toMatchSnapshot()
});
