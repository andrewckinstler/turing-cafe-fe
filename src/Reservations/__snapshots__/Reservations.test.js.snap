// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match a snapshot 1`] = `
<div>
  <ReservationCard
    date="12/29"
    id={1}
    key="1"
    name="Christie"
    number={12}
    time="7:00"
  />
  <ReservationCard
    date="4/5"
    id={2}
    key="2"
    name="Leta"
    number={2}
    time="7:00"
  />
</div>
`;
